### DISCUSSION TITLE
Feature: add sort by "random" filter in Browse - Album view

### DISCUSSION THREAD

**Discussion by rare-magma** [jcorporation/myMPD#829] [Ideas]
When opening the Browse - Album view it would be nice if the contents were different on every load, instead of always getting the same results ordered alphabetically by default.
In my opinion the Browse - Album view presents a good opportunity to become interested in an album accidentally by it's cover.
This doesn't happen if the same content is presented every time.

![image](https://user-images.githubusercontent.com/45106055/187528334-07927065-5be0-436c-bab6-713a60d673ba.png)

**Comment by jcorporation** [comment]
Duplicate of #457
